dist/index.html

```diff
@@ -126,28 +124,29 @@ <h2>Total Cost</h2>
     </section>
     <section class="pantry hidden" id="pantry-view">
       <section class="pantryContents">
-        <div class="dropDropMenu">
+        <menu class="dropDropMenu">
           <label for="ingredients">Add Ingredient to Pantry: </label>
             <select name="ingredients" id="ingredient-drop-down-menu" class="selectMenu">
             </select>
             <form class="quantity">
               <input type="text" placeholder="quantity" class="quantityInput" id="quantity-input">
             </form>
             <button type="button" class="addButton" id="add-button">Add to Pantry</button>
-        </div>
-        <div class="pantryItems">
-            <div class="ingredientsPantry">
-            <div class="pantryHeading">
+        </menu>
+        <article class="pantryItems">
+          <article class="ingredientsPantry">
+            <header class="pantryHeading">
```

Comment:
Would this be a header?

Reply:
I think a header is reserved for a header on the page. The <header> element represents a container for introductory content or a set of navigational links. A <header> element typically contains: one or more heading elements (<h1> - <h6>), logo/icon, or authorship information